docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md

```diff
@@ -0,0 +1,190 @@
+
+### User ID
+
+To get the User ID, follow the steps below:
+1. Login to **DocuSign** account
+2. Go to **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
+3. Locate and copy **User ID** available under **My Account Information**. <br/> <img src={useBaseUrl('img/send-data/docusign-user-id.png')} alt="<docusign-user-id.png>" width="600" />
+:::note
+You'll need to provide your DocuSign User ID while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
+
+### App
+
+You must first create an app to get integration key and configure RSA Key Pair and Redirect URI. This key is required to get access token which will be used to authenticate DocuSign API. To create an app follow the steps below:
+1. Login to **DocuSign** account
+2. Go to **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
+3. Navigate to **ADD APP AND INTEGRATION KEY**. <br/> <img src={useBaseUrl('img/send-data/docusign-add-app-integration-key.png')} alt="<docusign-add-app-integration-key.png>" width="600" />
+4. Enter value for **App Name** in a dialog box, and click on **CREATE APP**. <br/> <img src={useBaseUrl('img/send-data/docusign-create-app.png')} alt="<docusign-create-app.png>" width="600" />
+5. After app creation, you will be redirected to app configuration page. Copy **Integration Key**. <br/> <img src={useBaseUrl('img/send-data/docusign-integration-key.png')} alt="<docusign-integration-key.png>" width="600" />
+:::note
+Remember, you will need integration key while requesting [application consent](#app-consent) and creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
+7. Click on **GENERATE RSA** under **Service Integration** to create new private and public key pair. <br/> <img src={useBaseUrl('img/send-data/docusign-generate-rsa.png')} alt="<docusign-generate-rsa.png>" width="600" />
+8. Copy **Private Key** from dialog and close the dialog. <br/> <img src={useBaseUrl('img/send-data/docusign-private-key.png')} alt="<docusign-private-key.png>" width="600" />
+:::note
+Remember, you will need RSA private key while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
+9. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="600" />
+:::note
+Remember, you will need redirect uri while requesting [application consent](#app-consent).
+:::
+10. Click on **SAVE** to finish new app configuration.
+
+### App Consent
+
+For requesting application consent, follow below steps:
+1. Copy below url based on your account environment in browser and press enter, replace **INTEGRATION_KEY**, **REDIRECT_URI** with the values you copied at step 5 and step 8 of [App](#app) section respectively.
```

Comment:
The steps in the App section are out of order. Can you confirm "step 5 and step 8" are the correct ones you want to refer to?

Reply:
Have verified and step #s are correct.